cluster/images/kubelet/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM centos
```

Comment:
Maybe do something similar to what we do for nginx:

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/images/nginx/Dockerfile